test/templates/index.njk

```diff
@@ -107,8 +111,6 @@ li {
       <ul>
           <li><a href="pages/styles">Styles</a></li>
           <li><a href="pages/text">Text</a></li>
-          <li><a href="pages/form-elements">Forms</a></li>
-          <li><a href="pages/all">Sections and Panels</a></li>
```

Comment:
I'm happy to add these back and make them work with all the new form components if they are actively used. Seemed like a large amount of work if nobody was using them.